mailerlite.com.website.json

```diff
@@ -0,0 +1,33 @@
+{  
+   "providerId": "mailerlite.com",
+   "providerName": "MailerLite",
+   "serviceId": "website",
+   "serviceName": "MailerLite Sites",
+   "version": 1,
+   "description": "Create websites or landing pages using your own domain.",
+   "syncPubKeyDomain": "mailerlite.com",
+   "variableDescription": "Self explanatory in the variable names.",
+   "records": [
+      {
+         "groupId": "domain",
+         "type": "A",
+         "host": "@",
+         "pointsTo": "%ip%",
+         "ttl": 3600
+      },
+      {
+         "groupId": "redirect",
+         "type": "CNAME",
+         "host": "www",
+         "pointsTo": "%www_points_to%",
+         "ttl": 3600
+      },
+      {
+         "groupId": "subdomain",
+         "type": "CNAME",
+         "host": "%subdomain_host%",
```

Comment:
Do you intend to use this group always in combination with domain group, or standalone?
If standalone then a separate template would do better.